content/conditions/broken-or-bruised-ribs/manifest.json

```diff
@@ -0,0 +1,110 @@
+{
+  "layout": "content-simple",
+  "description": "Bruised or broken ribs can be very painful but usually heal by themselves.",
+  "choicesOrigin": "conditions/rib-injuries/pages/introduction.aspx",
```

Comment:
No, `http://www.nhs.uk/` gets applied in the [new site message template](https://github.com/nhsuk/betahealth/blob/develop/app/views/_includes/new-site-message.nunjucks#L9).